plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/BaseProjectTest.java

```diff
@@ -61,6 +61,11 @@ public void tearDown() {
 
       // ensure there are no jobs
       SwtBotWorkbenchActions.waitForProjects(bot, project);
+      {
+        String fileName = SWTBotPreferences.SCREENSHOTS_DIR + "/" + "post-test-" + project.getName()
+            + "." + SWTBotPreferences.SCREENSHOT_FORMAT.toLowerCase();
+        SWTUtils.captureScreenshot(fileName);
```

Comment:
Do you intend to always capture in a normal situation, or is this for a temporary testing?